var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -36,18 +36,19 @@ class PyNumpy(PythonPackage):
     homepage = "http://www.numpy.org/"
     url      = "https://pypi.io/packages/source/n/numpy/numpy-1.9.1.tar.gz"
 
+    version('1.12.0', '33e5a84579f31829bbbba084fe0a4300',
+            url="https://pypi.io/packages/source/n/numpy/numpy-1.12.0.zip")
     version('1.11.2', '03bd7927c314c43780271bf1ab795ebc')
     version('1.11.1', '2f44a895a8104ffac140c3a70edbd450')
     version('1.11.0', 'bc56fb9fc2895aa4961802ffbdb31d0b')
     version('1.10.4', 'aed294de0aa1ac7bd3f9745f4f1968ad')
     version('1.9.2',  'a1ed53432dbcd256398898d35bc8e645')
     version('1.9.1',  '78842b73560ec378142665e712ae4ad9')
 
-    variant('blas',   default=True)
-    variant('lapack', default=True)
+    variant('blas',   default=True, description='Build with BLAS support')
+    variant('lapack', default=True, description='Build with LAPACK support')
 
-    depends_on('python@2.6:2.8,3.2:')
-    depends_on('py-nose', type='build')
```

Comment:
Nose is not needed for installation. It is only used for testing purposes. We can add this back as a `test` deptype after #1279 and #2869 are complete.